Cargo.toml

```diff
@@ -28,6 +28,7 @@ cortex-m-log = { version = "~0.7", features = ["itm", "semihosting", "log-integr
 panic-itm = { version = "~0.4.2", optional = true  }
 panic-semihosting = { version = "0.5.6", optional = true  }
 cortex-m-semihosting = { version = "0.3.7", optional = true  }
+wm8731-alt = { git = "https://github.com/YruamaLairba/wm8731-alt", branch = "main" }
```

Comment:
I asked the author if he could publish it to crates.io. Alternatively we can use a [different crate](https://docs.rs/wm8731/latest/wm8731/).

Reply:
We can merge it like this for now. Can change it later as needed.